spec/requests/web/sessions_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe "Web::Sessions", type: :request do
```

Comment:
Is this spec auto-generated?
```suggestion
RSpec.describe Web::Sessions, type: :request do
```
There are also some double quoting instead of single quotes in the spec, but that's just 💅🏼

Reply:
Btw, this constant that you suggested, does not exist.